Build error locally and CI also reporting build errors

```
[100%] editor/editor_properties.cpp: In member function 'void EditorPropertyEnum::_option_selected(int)':
editor/editor_properties.cpp:396:63: error: invalid use of non-static member function 'StringName EditorProperty::get_edited_property()'
   emit_signal("property_changed", get_edited_property, p_which);
                                                               ^
In file included from ./editor/editor_plugin.h:34:0,
                 from ./editor/editor_help.h:34,
                 from ./editor/create_dialog.h:34,
                 from editor/editor_properties.h:34,
                 from editor/editor_properties.cpp:31:
./editor/editor_inspector.h:101:13: note: declared here
  StringName get_edited_property();
             ^~~~~~~~~~~~~~~~~~~
scons: *** [editor/editor_properties.x11.tools.64.o] Error 1
scons: building terminated because of errors.
```

--- a/editor/editor_properties.cpp
+++ b/editor/editor_properties.cpp
@@ -390,13 +390,37 @@ EditorPropertyCheck::EditorPropertyCheck() {
 
 void EditorPropertyEnum::_option_selected(int p_which) {
 
-	emit_signal("property_changed", get_edited_property(), p_which);
+	String text = options->get_item_text(p_which);
+	Vector<String> text_split = text.split(":");
+	if (text_split.size() == 1) {
+		emit_signal("property_changed", get_edited_property(), p_which);
+		return;
+	}
+	String name = text_split[1];
+	emit_signal("property_changed", get_edited_property(), name.to_int());
 }
 
 void EditorPropertyEnum::update_property() {
 
 	int which = get_edited_object()->get(get_edited_property());
-	options->select(which);
+	if (which == 0) {
+		options->select(which);
+		return;
+	}
+
+	for (int i = 0; i < options->get_item_count(); i++) {
+		String text = options->get_item_text(i);
+		Vector<String> text_split = text.split(":");
+		if (text_split.size() == 1) {
+			options->select(which);
+			return;
+		}
+		String name = text_split[1];
+		if (itos(which) == name) {
+			options->select(i);
+			return;
+		}
+	}
 }
 
 void EditorPropertyEnum::setup(const Vector<String> &p_options) {

